lib/theming/src/create.ts

```diff
@@ -170,7 +170,7 @@ export const convert = (inherit: ThemeVars = lightThemeVars): Theme => {
     brand: {
       title: brandTitle,
       url: brandUrl,
-      image: brandImage,
+      image: brandImage || brandTitle ? null : undefined,
```

Comment:
Doesn't this make brandImage never used? Shouldn't this be 

`image: brandImage || (brandTitle ? null : undefined),`

Reply:
Good catch 😖

Reply:
@kennethtruong Mind doing a small PR for this?

Reply:
Sure!